### DISCUSSION TITLE
Launch Options of "Xcom 2 Launcher"

### DISCUSSION THREAD

**Discussion by denisstepanov21011990** [X2CommunityCore/xcom2-launcher#369] [Help]
Where to view the list of available Launch Options?
	
Example: "D:\Xcom 2 Launcher.exe" --parametrs
What is the launch parameter that, after opening the AML and checking the versions, launches the game immediately?

**Comment by Neophewl** [comment]
bumping this. was hoping to just launch the game through playnight with a launch option

**Reply by RevZero** [reply]
If you just want to start the game with the currently configured mods, you could just launch XCOM directly. Once applied, the changes (enabled mods) from AML persist. So only launch AML if you want to make changes.
